gprofiler/__init__.py

```diff
@@ -2,4 +2,12 @@
 # Copyright (c) Granulate. All rights reserved.
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
+import sys
+from functools import lru_cache
+
 __version__ = "1.6.0"
+WINDOWS_PLATFORM_NAME = "win32"
```

Comment:
Let's move this code to `gprofiler/platform.py`? Then you can still import it early enough.

Reply:
Done